episodes/severity-static.Rmd

```diff
@@ -0,0 +1,788 @@
+onset_to_death_ebola <-
+  epiparameter::epidist_db(
+    disease = "Ebola",
+    epi_dist = "onset_to_death",
+    single_epidist = TRUE
+  )
+
+# Plot <epidist> object
+plot(onset_to_death_ebola, day_range = 0:40)
+```
+
+To calculate the delay-adjusted CFR, we can use the `cfr_static()` function with the `data` and `delay_density` arguments.
+
+```{r}
+# Calculate the static CFR while adjusting for delays
+cfr_static(
+  data = ebola1976,
+  delay_density = function(x) density(onset_to_death_ebola, x)
+)
+```
+
+```{r,echo=FALSE}
+out_delay_adjusted <-
+  cfr_static(
+    data = ebola1976,
+    delay_density = function(x) density(onset_to_death_ebola, x)
+  )
+
+out_mean <- out_delay_adjusted %>% pull(severity_mean)
+out_low <- out_delay_adjusted %>% pull(severity_low)
+out_high <- out_delay_adjusted %>% pull(severity_high)
+```
+
+The delay-adjusted CFR indicated that the overall disease severity _at the end of the outbreak_ or with the _latest data available at the moment_ is `r out_mean` with a 95% confidence interval between `r out_low` and `r out_high`, slightly higher than the naive one.
+
+::::::::::::::::::::::::::: spoiler
+
+### How does {cfr} works?
+
+To adjust the CFR, [Nishiura et al., 2009](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0006852) use the case and death incidence data to estimate the number of cases with known outcomes:
+
+$$
+  u_t = \dfrac{\sum_{i = 0}^t
+        \sum_{j = 0}^\infty c_{i - j} f_{j}}{\sum_{i = 0} c_i},
+$$
+
+where:
+
+- $c_{t}$ is the daily case incidence at time $t$, 
+- $f_{t}$ is the value of the Probability Mass Function (PMF) of the **delay distribution** between onset and death, and
+- $u_{t}$ represents the underestimation factor of the known outcomes.
+
+$u_{t}$ is used to **scale** the value of the cumulative number of cases in the denominator in the calculation of the CFR. This is calculated internally with the [`estimate_outcomes()`](https://epiverse-trace.github.io/cfr/reference/estimate_outcomes.html) function.
+
+The estimator for CFR can be written as: 
+
+$$p_{t} = \frac{b_{t}}{u_{t}}$$
+
+where $p_{t}$ is the realized proportion of confirmed cases to die from the infection (or the unbiased CFR), and $b_{t}$, the crude and biased estimate of CFR (also naive CFR).
+
+From this last equation, we observe that the unbiased CFR $p_{t}$ is larger than biased CFR $b_{t}$ because in $u_{t}$ the numerator is smaller than the denominator (note that $f_{t}$ is the probability distribution of the *delay distribution* between onset and death). Therefore, we refer to $b_{t}$ as the biased estimator of CFR.
+
+When we observe the entire course of an epidemic (from $t \rightarrow \infty$), $u_{t}$ tends to 1, making $b_{t}$ tends to $p_{t}$ and become an unbiased estimator ([Nishiura et al., 2009](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0006852)).
+
+:::::::::::::::::::::::::::
+
+:::::::::::::::::: callout
+
+### Use the `<epidist>` class
```

Comment:
here too the word epidist is very small and there's no spaces between words

Reply:
will fix #41